cli/src/pcluster/models/imagebuilder.py

```diff
@@ -427,26 +429,21 @@ def _check_instance_using_image(self):
                 return False
             raise ImageBuilderActionError(f"Unable to delete image and stack, due to {str(e)}")
 
-    def _validate_image_name(self):
-        match = re.match(r"^[-_A-Za-z-0-9][-_A-Za-z0-9 ]{1,126}[-_A-Za-z-0-9]$", self.image_name)
+    def _validate_id(self):
+        match = re.match(r"^([a-zA-Z][a-zA-Z0-9-]{0,127})$", self.id)
```

Comment:
can we have this in a constant? Is this the same as the one used to validate the cluster name?

Reply:
we can have this constant, but it is not same as cluster name, because the cluster name length is 60 chars, image length limit is 128 chars.